nodejs.org/test/run

```diff
@@ -101,6 +101,11 @@ test_s2i_usage() {
   s2i usage ${s2i_args} ${IMAGE_TAG} &>/dev/null
 }
 
+test_s2i_incremental() {
+  echo "Testing s2i build incremental..."
+  s2i build --incremental=true ${s2i_args} file://${test_dir}/test-app ${IMAGE_TAG} ${IMAGE_NAME}-testapp
```

Comment:
I see that you are rebuilding the image here, and you can verify from visual inspection that it's working. But it would be nice if there were an actual test to confirm the expected behavior. Maybe it's as simple as adding a `| grep "Restoring build artifacts"` to the end of this command and checking the output.